helm/alfresco-content-services/README.md

```diff
@@ -1,3 +1,354 @@
-# Alfresco Content Services (ACS) Helm Chart
+# alfresco-content-services
```

Comment:
Whilst I appreciate generating docs is a good thing to do this repo is used for end-user documentation at the moment and this overwrites one of the pages we need to guide users to the correct place. Furthermore, this information is already included in a different location.
We are starting a project with the docs team to address this but in the meantime I need to keep the docs as they are and address the auto-generated docs in https://issues.alfresco.com/jira/browse/OPSEXP-435

Reply:
that README is basically empty ATM though and if you move that info in the description field of Chart.yaml you have it there so I think we could go for it

helm-docs is less smart than terraform-docs as it overwrites with a template rather than using a placeholder but maybe it's in the docs and I missed that

Reply:
Yes, we have the table of configuration options on this page: https://github.com/Alfresco/acs-deployment/blob/master/docs/helm/README.md so I don't really want it duplicated. 
Can we disable the docs generation for now and do it in OPSEXP-435?